pierone/cli.py

```diff
@@ -242,22 +210,28 @@ def tags(config, team: str, artifact, url, output, limit):
 
     if not artifact:
         artifact = get_artifacts(config.get('url'), team, token)
+        if not artifact:
+            raise click.UsageError('Either Team does not exist or currently does not artifacts in Pierone! '
```

Comment:
Users keep asking me what is wrong then there is nothing and error code in command line is 0 when it should be 1. I think it is a good practice to advise the user what might be wrong when they do not see anything.